.ci/download-deps.sh

```diff
@@ -3,8 +3,8 @@
 
 function get_deps() {
 
-	TSS_VERSION="2.3.1"
-	ABRMD_VERSION="2.3.0"
+	TSS_VERSION=master
+	ABRMD_VERSION=master
```

Comment:
Do we need these set to master or can we bump them to a stable release for the feature you need?

Reply:
@williamcroberts the change for retrieving the sapi handle is in tpm2-tss master. I also see it is in v2.4rc1. Perhaps I can tag this as 2.4 and wait until the rc becomes a full release.